lib/std/os/windows.zig

```diff
@@ -1603,6 +1604,24 @@ pub fn CreateProcessW(
             .INVALID_PARAMETER => unreachable,
             .INVALID_NAME => return error.InvalidName,
             .FILENAME_EXCED_RANGE => return error.NameTooLong,
+            // These are all the system errors that are mapped to ENOEXEC in ReactOS:
+            // https://doxygen.reactos.org/d8/d68/doserrmap_8h_source.html
```

Comment:
nit: add the commit and shorten things abit, for example with
`System errors mapped to ENOEXEC in ReactOS as of 0.4.15-dev-5519-ge8e38b8`.
This should probably link to the webpage for the releases.

Reply:
Ended up digging deeper to understand where that mapping was coming from and updated the comment to be much more specific.